pkg/nsx/services/securitypolicy/store.go

```diff
@@ -1,153 +1,92 @@
 package securitypolicy
 
 import (
-	"fmt"
-	"strconv"
-	"strings"
-	"sync"
-
 	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	"k8s.io/client-go/tools/cache"
 
 	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
-	nsxutil "github.com/vmware-tanzu/nsx-operator/pkg/nsx/util"
-	"github.com/vmware-tanzu/nsx-operator/pkg/util"
 )
 
-func queryTagCondition(service *SecurityPolicyService) string {
-	return fmt.Sprintf("tags.scope:%s AND tags.tag:%s",
-		strings.Replace(util.TagScopeCluster, "/", "\\/", -1),
-		strings.Replace(service.NSXClient.NsxConfig.Cluster, ":", "\\:", -1))
+// keyFunc is used to get the key of a resource, usually the ID
```

Comment:
you can rephrase is to "usually, which is the ID of the resource"

Reply:
ok